pkg/kubectl/cmd/apiversions.go

```diff
@@ -51,19 +51,13 @@ func RunApiVersions(f *cmdutil.Factory, w io.Writer) error {
 		return err
 	}
 
-	apiVersions, err := client.ServerAPIVersions()
+	groupList, err := client.Discovery().ServerGroups()
 	if err != nil {
 		fmt.Printf("Couldn't get available api versions from server: %v\n", err)
 		os.Exit(1)
 	}
+	apiVersions := unversioned_client.ExtractGroupVersions(groupList)
 
-	var expAPIVersions *unversioned.APIVersions
-	expAPIVersions, err = client.Extensions().ServerAPIVersions()
-
-	fmt.Fprintf(w, "Available Server Api Versions: %#v\n", *apiVersions)
-	if err == nil {
-		fmt.Fprintf(w, "Available Server Experimental Api Versions: %#v\n", *expAPIVersions)
-	}
-
+	fmt.Fprintf(w, "Available Server API Versions: %v\n", apiVersions)
```

Comment:
For human readable output, I think they'd prefer the strings joined or on separate lines and there should be a guaranteed sort.  The server side may be guaranteeing an order, but its not obvious from the client docs.